docs/getting-started/install-endpoint.asciidoc

```diff
@@ -131,15 +141,51 @@ Behavioral ransomware prevention detects and stops ransomware attacks on Windows
 
 Ransomware protection is a paid feature and is enabled by default if you have a https://www.elastic.co/pricing[Platinum or Enterprise license]. If you upgrade to a Platinum+ license from Basic or Gold, ransomware protection will be disabled by default.
 
-Ransomware protection levels are as follows:
+Ransomware protection levels are:
 
-* **Detect**: Detects ransomware on the host and generates an alert. The {agent} will **not** block malware. Select the **Notify User** option to enable user notifications.
+* **Detect**: Detects ransomware on the host and generates an alert. The {agent} will **not** block ransomware. Select the **Notify User** option to enable user notifications.
 * **Prevent** (Default): Detects ransomware on the host, blocks it from executing, and generates an alert. User notifications are enabled by default. Deselect the **Notify User** option to disable them.
 +
 TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {filename}` syntax.
 
 [role="screenshot"]
-image::images/install-endpoint/ransomware-protection.png[]
+image::images/install-endpoint/ransomware-protection.png[Detail of ransomware protection section.]
+
+[discrete]
+[[memory-protection]]
+=== Memory manipulation protection
+
+Memory manipulation protection detects and stops in-memory threats on Windows systems, such as shellcode injection, which are used to evade traditional file-based detection techniques.
+
+Memory manipulation protection is a paid feature and is enabled by default if you have a https://www.elastic.co/pricing[Platinum or Enterprise license]. If you upgrade to a Platinum+ license from Basic or Gold, memory manipulation protection will be disabled by default.
+
+Memory manipulation protection levels are:
+
+* **Detect**: Detects memory manipulation activity on the host and generates an alert. The {agent} will **not** block the in-memory activity. Select the **Notify User** option to enable user notifications.
+* **Prevent** (Default): Detects memory manipulation activity on the host, blocks it from executing, and generates an alert. User notifications are enabled by default. Deselect the **Notify User** option to disable them.
```

Comment:
This should be changed as well like https://github.com/elastic/security-docs/pull/994/files#r706362168

Here's my suggestion

```
Detects memory manipulation activity on the host, kills the offending process or thread, and generates an alert.
```

cc @joe-desimone @gabriellandau @bradenlpreston @caitlinbetz